man/rules/meson.build

```diff
@@ -824,6 +824,7 @@ manpages = [
  ['systemd.kill', '5', [], ''],
  ['systemd.link', '5', [], ''],
  ['systemd.mount', '5', [], ''],
+ ['systemd.naming-scheme', '7', [], ''],
```

Comment:
hmm, that's a bit generic a name. can we rename this to "systemd.net-naming-scheme" or so? i.e. include "net" or so in the name, to clarify this is about network interface naming, and nothing else?

Reply:
Yeah, I'm not too happy about the name. I think `net.naming-scheme` was an error, it should have been `udev.net-naming-scheme` or something. (And `udev.ifnames`, not `net.ifnames`.)

I'll change it to `systemd.net-naming-scheme` because that seems least bad.